### DISCUSSION TITLE
Advice on Updating from WCAG 3 Draft APCA 0.98G 4g to the Latest Beta Version

### DISCUSSION THREAD

**Discussion by UXPublishing** [Myndex/SAPC-APCA#100] [General]
In July 2021, I used the WCAG 3 Draft APCA 0.98G 4g formulas and constants in Excel to create a contrast tool. I am trying to determine what, if anything, I need to update in my tool to match the beta version. 
Do I need to:
a) change any formulas?
b) change any constants?

Should I wait to make any updates until after everything is finalized (out of beta)?

For more information about what version I used and my tool, you can read this closed issue. https://github.com/Myndex/SAPC-APCA/issues/12

**Comment by Myndex** [comment]
Hi @UXPublishing 

The 0.0.98G 4g constants are the current constants. The apca-w3 version has a simplification on the output where SAPC < 0.1 is clamped to 0 see the apca-w3 repo, or the npm apca-w3

There will be another version with some enhanced features at some point, currently in private beta and undergoing study, but for now the 98G 4g series constants are the current public beta.

As far as waiting: do you have a tool you are wanting to release in some way?

**Comment by UXPublishing** [comment]
Thanks for clarifying! 

When might the "final" version be released?

I am not in a rush. Right now, I am just using my spreadsheet tool to define accessible stylesheets for web and desktop publishing projects. For example, I am defining all the text and color style guidelines for my ebooks and websites. 

If the constants and formulas have not changed since July 2021, then my tool is up to date. The only thing I would have to update is the Font to Contrast lookup tables, which has changed since July 2021.

**Reply by Myndex** [reply]
I am trying to have a version with new features Sept 6th... (colorblind awareness day)

**Reply by UXPublishing** [reply]
Thank you for the update!
